### PR TITLE
DDF-4268 Switches ui test runner to headless chrome

### PR DESCRIPTION
**Pull request by djblue**
#### What does this PR do?

Switches ui test runner to headless chrome

#### Who is reviewing it? 

@adimka 
@gjvera  

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler 
@rzwiefel

#### How should this be tested?

- Successful CI build

#### What are the relevant tickets?

[DDF-4268](https://codice.atlassian.net/browse/DDF-4268)

### CODE REVIEW

File: ui/packages/ace/lib/test.js [hunk 1]
```diff
@@ -1,22 +1,18 @@
...
-    { stdio: 'inherit' }
-  )
+const { runner } = require('mocha-headless-chrome')
 
-  ps.on('exit', process.exit)
+module.exports = ({ args, pkg }) => {
+  runner({ file: args, reporter: 'spec' })
+    .then(({ result, coverage }) => {
+      if (result.stats.failures) {
+        throw 'Tests fails'
```
Comment (adimka) [inline ui/packages/ace/lib/test.js]: ✏️ Tests failed?

File: ui/packages/ace/lib/test.js [hunk 2]
```diff
@@ -1,22 +1,18 @@
-const { spawn } = require('child_process')
-const phantomjs = require('phantomjs-prebuilt')
+const fs = require('fs')
+const path = require('path')
```
Comment (willwill96) [inline ui/packages/ace/lib/test.js]: ❓ Should this be here? It doesn't look like you use `path` anywhere

**Comment by andrewkfiedler** [comment]
🚫 until we update the CI to handle these changes

**Comment by LinkMJB** [comment]
Docker image supporting CI builds has been updated to support this.
build now

**Comment by rzwiefel** [comment]
## Windows builds successfully 🎉 
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
```

**Pull request merged**